src/js/components/CollapsingString.js

```diff
@@ -118,12 +118,13 @@ class CollapsingString extends React.Component {
 
   render() {
     let fullString = this.props.string;
+    let endLength = this.props.endLength || Math.floor(fullString.length * 1/3);
```

Comment:
not sure this might ever happen, but what if `props.endLength` is 0? i can imagine us sometime in the far future programmatically changing `endLength` to 0 for some reason.

sorry for the nit